utilities.ts

```diff
@@ -20,8 +20,8 @@ var inBrowser = typeof console !== "undefined" && console.info;
 declare var putstr;
 declare var printErr;
 
-/** @const */ var release: boolean = true;
-/** @const */ var profile: boolean = true;
+/** @const */ var release: boolean = jsGlobal.config.release;
+/** @const */ var profile: boolean = !release;
```

Comment:
Yes, there is quite a bit of code that is removed by closure if it can prove that `release` is false, it's not just assertion code.